dubbo-common/src/main/java/org/apache/dubbo/common/config/Configuration.java

```diff
@@ -67,7 +73,15 @@
      * @return the value to which this configuration maps the specified key, or default value if the configuration
      * contains no mapping for this key.
      */
-    Object getProperty(String key, Object defaultValue);
+    default Object getProperty(String key, Object defaultValue) {
+        Object value = getInternalProperty(key);
+        if (value == null) {
+            value = defaultValue;
+        }
+        return value;
```

Comment:
can be simplified with 
`return value!=null? value : defaultValue?`

Reply:
done